jfrog-applications/jfrog-cli/cli-command-summaries.md

```diff
@@ -0,0 +1,150 @@
+# CLI Command Summaries
+
+## Overview
+
+The command summaries feature enables the recording of JFrog CLI command outputs into the local file system.
```

Comment:
The command summaries --> The **Command Summaries**

(notice the upper cases and thr bold style when mentioning the name of the featire. Let's implement this guideline throughout the page, if the feature name is mentioned again).